apps/transport/lib/transport_web/templates/dataset/_resource.html.eex

```diff
@@ -94,17 +94,22 @@
   <% end %>
   <%= if Resource.has_errors_details?(@resource) do %>
     <% nb_errors = errors_count(@resource) %>
-    <%= if nb_errors do %>
+    <% nb_warnings = warnings_count(@resource) %>
+    <%= unless is_nil(nb_errors) and is_nil(nb_warnings) do %>
```

Comment:
si nb_errors vaut nil, mais pas nb_warnings...

Reply:
Le `and` devrait être prioritaire, non ? Il faut des parenthèses ?

Reply:
```
iex(2)> unless is_nil(nil) and is_nil(nil) do
...(2)>    IO.puts "Condition is not satisfied"
...(2)> else
...(2)>    IO.puts "else"
...(2)> end
else
:ok
```

Reply:
Si je comprends bien, `Resource.has_errors_details?(@resource)` nous assure que error_counts n'est pas nul. Du coup on peut enlever le check sur `is_nil(nb_errors)` de la ligne 98 ?

Reply:
La scénario qui m'embetait c'était le `unless is_nil(nil) and is_nil(3)`, mais visiblement le premier ne peut pas être nil.